src/content/guidelines/iconography/usage.mdx

```diff
@@ -41,19 +34,19 @@ tabs: ['Library', 'Usage', 'Contribution']
 <br />
 <br />
 
-## Usage for designers
+## Designing with icons
 
 ### Sizing
 
-UI icons that appear within Carbon components are generally 16 px squares, however icons sized at 20 px, 24 px, and 32 px can be also used within the UI. Icons should only be used at their original sizes and not resized in code.
+Carbon components typically use icons on 16px artboards. Icons on 20px, 24px, and 32px artboards can also be used within the UI.
```

Comment:
Maybe add something about being consistent with icons sizes across your product. A single product probably should be using 16, 20, 24, and 32.

Reply:
Good point! I'll add and request your review 🙌